Create new rule generate insightful reports via company emails (#7576)
SSWConsulting/SSW.Rules.Content: rules/generate-insightful-reports-via-company-emails/rule.md

---
type: rule

> title: Do you generate insightful reports via company emails?

Reviewer: ```suggestion
title: Do you gather data from company emails?
```
Consider changing the title to something more simple like this

> uri: generate-insightful-reports-via-company-emails

Reviewer: Update the URI to match the title

> uri: generate-insightful-reports-via-company-emails

Reviewer 2: Please also add this rule to a category. Checkout: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Add-and-Edit-Categories-and-Top-Categories

authors:
  - title: Isabel Sandstroem
    url: https://www.ssw.com.au/people/isabel-sandstroem/
created: 2023-12-15T01:24:13.247Z
guid: 9477fb4c-b872-4c82-af96-34d1cabd1d7c
---

> The emails exchanged within your company are more than just communication tools; they are a rich source of data. By analyzing this data, we can gain insights into various aspects of our business operations.

Reviewer: ```suggestion
The emails exchanged within your company are more than just communication tools; they are a rich source of data. By analyzing this data, you can gain insights into a company's operations.
```

<!--endintro-->

> At SSW, we follow a set of [email rules](www.ssw.com.au/rules/rules-to-better-email/) that ensure our emails are consistent and structured. This consistency is crucial for effective data analysis, as it standardizes the format and content of the emails, making them easier to parse and understand.  

Reviewer: ```suggestion
Emails should follow a set of [email rules](www.ssw.com.au/rules/rules-to-better-email/) that make communication consistent and structured. When emails have standard format and content, data analysis is possible because the content can be parsed.  
```

> With **Microsoft Graph API** we are able to extract Outlook data. The extracted data is not just collected; it is transformed into actionable insights through Power BI reports. These reports provide a visual representation of the data, making it easier to interpret and use for strategic decision-making.

Reviewer: ```suggestion
**Microsoft Graph API** can be used to extract Outlook data. The extracted data is not just collected; it is transformed and displayed in Power BI reports. These reports provide a visual representation of the data, making it easier to interpret and use for strategic decision-making.
```

> ::: img-large
> ![](eagleeyepbireport.jpg)
> :::
> ::: good
> Figure: Good example - Visual representation in Power BI highlighting the insights from the rule [Do you use 'Checked by xxx'?](https://www.ssw.com.au/rules/checked-by-xxx/)
> :::

Reviewer: ```suggestion
::: good
![Figure: Good example - Visual representation in Power BI highlighting the insights from the rule [Do you use 'Checked by xxx'?](https://www.ssw.com.au/rules/checked-by-xxx/)](eagleeyepbireport.jpg)
:::
```
